rvgo/cmd/log.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/log"
```

Comment:
As a heads up, go-ethereum has just pulled in upstream geth changes which changes the logging library. https://github.com/ethereum-optimism/optimism/pull/8917 updated the monorepo for the new logging changes.

Doesn't need to be handled in this PR but probably is worth pulling in the updated op-geth soon.